metrics/micrometer/revapi.json

```diff
@@ -0,0 +1,22 @@
+// Configures Revapi (https://revapi.org/getting-started.html) to check API compatibility between
+// successive driver versions.
+{
+  "revapi": {
+    "java": {
+      "filter": {
+        "packages": {
+          "regex": true,
+          "exclude": [
+            "com\\.datastax\\.oss\\.protocol\\.internal(\\..+)?",
+            "com\\.datastax\\.(oss|dse)\\.driver\\.internal(\\..+)?",
+            "com\\.datastax\\.oss\\.driver\\.shaded(\\..+)?",
+            "com\\.datastax\\.oss\\.simulacron(\\..+)?",
+            // Don't re-check sibling modules that this module depends on
+            "com\\.datastax\\.(oss|dse)\\.driver\\.api\\.core(\\..+)?",
+            "com\\.datastax\\.(oss|dse)\\.driver\\.api\\.querybuilder(\\..+)?"
+          ]
+        }
+      }
+    }
+  }
+}
```

Comment:
The two new modules only contain internal classes, so this won't match anything. We can simply skip the plugin in `pom.xml`:

```xml
      <plugin>
        <groupId>org.revapi</groupId>
        <artifactId>revapi-maven-plugin</artifactId>
        <configuration>
          <skip>true</skip>
        </configuration>
      </plugin>
```